### PR TITLE
Connectomics module

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Next step in the refactoring (and to the preparation of the scil retreat): Managing scripts using connectivity. Once again, no code was changed, only moved.

Now, the main are empty (and more-or-less proof-read), for scripts:
- scil_print_connectivity_filenames.py 
- scil_compare_connectivity.py 
- scil_reorder_connectivity.py 
- scil_evaluate_connectivity_graph_measures.py 
- scil_normalize_connectivity.py     (one line of code was changed: remove the copy(in_matrix) as discussed on teams)

Some functions are placed in connectivity.utils (renamed connectivity.connectivity_tools as it has become more important), others are in stats.matrix_stats. 

Unit tests are not created. 

Scripts that remain because they are more complex and can be managed later:
- scil_compute_connectivity.py
- scil_decompose_connectivity.py
- scil_save_connections_from_hdf5.py
- scil_connectivity_math.py
- scil_filter_connectivity.py 

------
Question: import bct: not listed in requirements. I added something like Charles did for pyopencl. Ok?

Remark: pytest warnings:
- reorder_connectivity script: ClusterWarning: scipy.cluster: The symmetric non-negative hollow observation matrix looks suspiciously like an uncondensed distance matrix. Ok?
- evaluation_connectivity_graph_measures: RuntimeWarning: invalid value encountered in double_scalars. (dans omega_..) ok?

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by arnaudbore — APPROVED** [review]
LGTM waiting for @frheault review before merging

File: scilpy/connectivity/connectivity_tools.py
Comment (AlexVCaron) [inline scilpy/connectivity/connectivity_tools.py:67]: Move this to IO

File: scripts/scil_normalize_connectivity.py
Comment (AlexVCaron) [inline scripts/scil_normalize_connectivity.py:135]: Place assert at top of the script to prevent processing if the files aren't available

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
Most of the new files are missing a newline at the end. The should also be a newline between the doctring and the beginning of the code in the functions.

Maybe we let that as is for now and auto-reformat the whole scilpy codebase with autopep8 or something when the whole refactoring is done. We have to configure pep8 bettter so the code standards are better enforced.

**Review by frheault — APPROVED** [review]
I like how it is organized! It is ready for Monday (after Alex comment)

**Comment by EmmaRenauld** [comment]
> Most of the new files are missing a newline at the end.

I don't see that they are missing... Github's pep8 would have told me (I'm usually indeed forgetting them!) Github hides them in the preview I think.

> There should also be a newline between the doctring and the beginning of the code in the functions.

That's not done in most files so yes, let's keep it for later.

**Review by AntoineTheb — APPROVED** [review]
I see no glaring style errors and have not verified the code as its only being moved around. LGTM then.

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**
